sw/device/rom_exts/rom_ext_slot_b.ld

```diff
@@ -0,0 +1,33 @@
+/* Copyright lowRISC contributors. */
+/* Licensed under the Apache License, Version 2.0, see LICENSE for details. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/**
+ * Linker script for an OpenTitan ROM_EXT.
+ *
+ * Portions of this file are Ibex-specific.
+ *
+ * The ROM_EXT is actually kept in flash, rather than ROM. While a ROM_EXT can
+ * be loaded into either Slot A (the start of flash), or Slot B (the start of
+ * the upper half of flash), this linker script only targets Slot B.
+ */
+
+OUTPUT_ARCH(riscv)
+GROUP(-lgcc)
+
+/**
+ * Indicate that there are no dynamic libraries, whatsoever.
+ */
+__DYNAMIC = 0;
```

Comment:
I don't think these have to come before the contents afterwards, so they can move to the common linker script. The same applies to reserving space for the stack, I think.

Reply:
Done.